istioctl/cmd/metrics.go

```diff
@@ -137,8 +137,9 @@ func run(c *cobra.Command, args []string) error {
 	printHeader(c.OutOrStdout())
 
 	workloads := args
+	mDuration := int64(metricsDuration.Seconds())
```

Comment:
I'm not sure why need this, prometheus can not handle `1m0s` format?
BTW, `mDuration` is not that good, maybe you can change it to `durationSeconds` or something like?

Reply:
I'm sure prometheus not handle 1m0s format, only 1m 1s 1h ...（Verified）

thanks, `durationSeconds` is a good idea

Reply:
what version `prometheus` you're using?

there's no `Error executing query` on my env
```
2022-03-10T03:04:49.151796Z	debug	executing query: sum(rate(istio_requests_total{destination_workload=~"httpbin.*", destination_workload_namespace=~".*",reporter="destination"}[2m0s]))  result:
2022-03-10T03:04:49.151828Z	debug	no values for query: sum(rate(istio_requests_total{destination_workload=~"httpbin.*", destination_workload_namespace=~".*",reporter="destination"}[2m0s]))
2022-03-10T03:04:49.153206Z	debug	executing query: sum(rate(istio_requests_total{destination_workload=~"httpbin.*", destination_workload_namespace=~".*",reporter="destination",response_code=~"[45][0-9]{2}"}[2m0s]))  result:
2022-03-10T03:04:49.153228Z	debug	no values for query: sum(rate(istio_requests_total{destination_workload=~"httpbin.*", destination_workload_namespace=~".*",reporter="destination",response_code=~"[45][0-9]{2}"}[2m0s]))
2022-03-10T03:04:49.154555Z	debug	executing query: histogram_quantile(0.500000, sum(rate(istio_request_duration_milliseconds_bucket{destination_workload=~"httpbin.*", destination_workload_namespace=~".*",reporter="destination"}[2m0s])) by (le))  result:
2022-03-10T03:04:49.154578Z	debug	no values for query: histogram_quantile(0.500000, sum(rate(istio_request_duration_milliseconds_bucket{destination_workload=~"httpbin.*", destination_workload_namespace=~".*",reporter="destination"}[2m0s])) by (le))
2022-03-10T03:04:49.155774Z	debug	executing query: histogram_quantile(0.900000, sum(rate(istio_request_duration_milliseconds_bucket{destination_workload=~"httpbin.*", destination_workload_namespace=~".*",reporter="destination"}[2m0s])) by (le))  result:
2022-03-10T03:04:49.155796Z	debug	no values for query: histogram_quantile(0.900000, sum(rate(istio_request_duration_milliseconds_bucket{destination_workload=~"httpbin.*", destination_workload_namespace=~".*",reporter="destination"}[2m0s])) by (le))
2022-03-10T03:04:49.157157Z	debug	executing query: histogram_quantile(0.990000, sum(rate(istio_request_duration_milliseconds_bucket{destination_workload=~"httpbin.*", destination_workload_namespace=~".*",reporter="destination"}[2m0s])) by (le))  result:
2022-03-10T03:04:49.157182Z	debug	no values for query: histogram_quantile(0.990000, sum(rate(istio_request_duration_milliseconds_bucket{destination_workload=~"httpbin.*", destination_workload_namespace=~".*",reporter="destination"}[2m0s])) by (le))

```

Reply:
Oh, I I forgot to consider version problem, I used the v2.16, This version is too low.